docs/release-process.md

```diff
@@ -0,0 +1,173 @@
+
+```
+type(scope): description
+```
+
+The `type` field describes the nature of changes you are making. This project requires the type to be one of these exact names (bold names are preferred in most cases):
+
+* **`feat`**: Adding a new feature. (Will cause a minor version bump.)
+  * Use an `!` immediately before the `:` to signal an API breaking change. Once we release 1.0, this will cause a major version bump. Also note that for library crates, release-plz will independently evaluate all changes to the API surface and call for minor or major version bumps as appropriate.
+* **`fix`**: Bug fix. (Will cause a patch version bump.)
+* **`chore`**: Maintenance work. (Will not trigger a release PR and will not be included in changelog.)
+* **`docs`**: Revising documentation.
+* `build` or `ci`: Adjusting the build system.
+* `perf`: Performance optimization.
+* `refactor`
+* `revert`
+* `style`
+* `test`
+* `update`: Updating a dependency. (Used by Dependabot.)
+
+The `scope` field describes where the change is made. This project allows the scope to be omitted, but if it is present, it must be one of these exact names:
+
+* `c2patool`
+* `cawg_identity`
+* `crypto`
+* `export_schema`
+* `make_test_images`
+* `sdk`
+
+If `scope` is omitted, the parenthesis must also be omitted.
+
+`description` is a short human-readable summary of the changes being made. It is required. We prefer that the `description` be less than 70 characters and will issue a warning (which you can ignore) if it is longer.
+
+The following items are not enforced, but we ask that you observe the following preferences in `description`:
+
+* The entire description should be written and capitalized as an English-language sentence, except (as noted earlier) that the trailing period must be omitted.
+* Any acronyms such as JSON or YAML should be capitalized as per common usage in English-language sentences.
+
+The "body" of the commit message (everything after the PR title) is not subject to any restrictions and may be empty. GitHub, by default, will create a bullet list of the commits that went into the PR. It is _recommended,_ but not enforced, that you delete this list (because it typically contains a lot of signal noise) and either replace it with additional context of why you made the change or leave it empty.
+
+MAINTENANCE NOTE: If this list of rules is changed, please keep in sync with [`.github/workflows/pr_title.yml`](../.github/workflows/pr_title.yml) and [`.commitlintrc`](../.commitlintrc).
+
+## Nightly build process
+
+We maintain a preflight build tree which is designed to allow downstream projects (notably, language bindings) to verify compatibility with SDK changes before they are pushed to a formal release. We call these "nightly" builds, but in reality there are several triggers which can cause a nightly build to be created:
+
+* Any push to `main` branch
+* Any release pull request
+* 0500 UTC every day (typically overnight in North American time zones)
```

Comment:
I'm not sure if we are getting any value out of the automated overnight builds. LMK if it has been helpful.

Reply:
See #1065.